arch/x86/README.md

```diff
@@ -0,0 +1,2 @@
+x86 Architecture Support Components
+===================================
```

Comment:
This would be a good place to specify what's supported (e.g. that it's 32 bit) as well as any caveats or specific decisions (which features are used, any relationship to specific x86 variants, such as 486 vs 386 vs ...), perhaps specifically how segmentation vs. virtual memory are used in this implementation.